emukit/bayesian_optimization/acquisitions/expected_improvement.py

```diff
@@ -88,3 +90,338 @@ def get_standard_normal_pdf_cdf(x: np.array, mean: np.array, standard_deviation:
     pdf = scipy.stats.norm.pdf(u)
     cdf = scipy.stats.norm.cdf(u)
     return u, pdf, cdf
+
+
+class QExpectedImprovement(ExpectedImprovement):
+    def __init__(self, model: Union[IModel, IDifferentiable, IJointlyDifferentiable], jitter: float = float(0),
+                 fast_compute: bool = False, eps: float = 1e-3) -> None:
```

Comment:
no need for float(0), just 0.0 is fine. also, i believe there shouldn't be space around equals when defining default values.

Reply:
The spaces are there because ExpectedImprovement-class has them and I didn't want to edit the existing code just for style issues. Should I remove the spaces also from ExpectedImprovement?

Reply:
If that is so - yes, please, that would be great to fix in EI file too. Thanks!

Reply:
Fixed now